src/rendering/batcher/shared/Batcher.ts

```diff
@@ -212,7 +212,7 @@ export class Batcher
         // ++BATCH_TICK;
         const elements = this._elements;
 
-        let textureBatch = this._textureBatchPool[this._textureBatchPoolIndex++] || new BatchTextureArray();
+        let textureBatch = this._textureBatchPool[this._textureBatchPoolIndex++] ||= new BatchTextureArray();
```

Comment:
I once read somewhere that out-of-bounds array access and assignment permanently slow down the array in Chrome, and that it is therefore is not recommended. I've seen this in other places in PIXI's code base, so I thought it might be worth bringing up.

Reply:
thats a good point! Il add a follow up task to investigate this in all areas